easybuild/easyconfigs/r/Ray-assembler/Ray-assembler-2.3.1-iimpi-2019a.eb

```diff
@@ -15,7 +15,7 @@ toolchain = {'name': 'iimpi', 'version': '2019a'}
 toolchainopts = {'usempi': True}
 
 source_urls = [('https://sourceforge.net/projects/denovoassembler/files/', 'download')]
-sources = [SOURCE_TAR_BZ2]
+sources = ['ray-%(version)s.tar.bz2']
```

Comment:
@Darkless012 This should be `Ray-%(version)s.tar.bz2`?

Reply:
Thanks for pointing out